flutter/shell/platform/tizen/tizen_view_elementary.cc

```diff
@@ -367,6 +367,10 @@ void TizenViewElementary::Show() {
   evas_object_show(image_);
 }
 
+void TizenViewElementary::UpdateFlutterCursor(const std::string& kind) {
+  FT_LOG(Error) << "UpdateFlutterCursor is not supported.";
```

Comment:
This will result in too verbose error output on non-TV devices. I think the log level can be lowered to Info since the cursor type support is usually not that critical.

Reply:
Changed to `Info`.